src/bgnbd_expectation.cpp

```diff
@@ -0,0 +1,82 @@
+#include <RcppArmadillo.h>
+#include <math.h>
+#include "bgnbd_expectation.h"
+#include "clv_vectorized.h"
+
+//' @name bgnbd_expectation
+//' @title BG/NBD: Unconditional Expectation
+//'
+//' @description Computes the expected number of repeat transactions in the interval (0, vT_i]
+//' for a randomly selected customer, where 0 is defined as the point when the customer came alive.
```

Comment:
move to template and re-use in all rcpp expectation functions (bgnbd, pnbd, ggomnd)